src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,27 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    private final Bulldozer bulldozerD4;
+    private final Bulldozer bulldozerD5;
+    private final Bulldozer bulldozerD6Lgp;
+
+    public BulldozerProducer() {
+        bulldozerD4 = new Bulldozer("VPAT", 3.81, 14008);
+        bulldozerD4.setName("Cat D4");
+        bulldozerD4.setColor("Yellow");
+        bulldozerD5 = new Bulldozer("Semi-Universal", 4.3, 17340);
+        bulldozerD5.setName("Cat D5");
+        bulldozerD5.setColor("Yellow");
+        bulldozerD6Lgp = new Bulldozer("Semi-Universal", 5.8, 22870);
+        bulldozerD6Lgp.setName("Cat D6");
+        bulldozerD6Lgp.setColor("Yellow");
+    }
```

Comment:
```suggestion
```
Let's not create separate fields for your instances. I'd recommend directly creating them in get() method. Also you can pass parameters directly to the constructor so that setters are not used. Please fix the same issue in all places